internal/lambdalabs/v1/capabilities.go

```diff
@@ -30,3 +30,12 @@ func (c *LambdaLabsClient) GetCapabilities(_ context.Context) (v1.Capabilities,
 
 	return capabilities, nil
 }
+
```

Comment:
can we find a way to combine client + credential capabilities these? also is appears lambda does not support modify firewall since it is project level